test/scenarios/el94-src@low-latency.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export SKIP_GREENBOOT=true
```

Comment:
Do we need [this](https://github.com/openshift/microshift/pull/3571/files#diff-7930c5d05c02d3e1a79836ba848d5ba4bd41fa0a1c9858143af69c1f1599d485R61) if we are skipping greenboot checks?

Reply:
Later in test the MicroShift gets enabled and we run greenboot now.